lib/config/env.js

```diff
@@ -123,6 +123,9 @@ const ENV_MAPPING = {
       enabled: 'NEW_RELIC_DATASTORE_DATABASE_NAME_REPORTING_ENABLED'
     }
   },
+  grpc: {
```

Comment:
you're going to need open a PR in docs-website to capture these new config options

Reply:
Will do.